lib/omnibus/health_check.rb

```diff
@@ -124,6 +124,12 @@ class HealthCheck
       /libgcc_s.so/,
       /libstdc\+\+\.so/,
       /libcrypt.so/,
+      /libcrypto.so/,
+      /libssl.so/,
```

Comment:
These are an unfortunate side-effect of Solaris 11's Cryptographic Framework, which kind of includes ssl in the kernel. If you're morbidly curious: https://docs.oracle.com/cd/E23824_01/html/821-1456/scf-10.html#scrolltoc